src/AutoPilotPlugins/APM/APMHeliComponent.cc

```diff
@@ -23,7 +23,7 @@ QString APMHeliComponent::name(void) const
 
 QString APMHeliComponent::description(void) const
 {
-    return tr("Heli Setup is used to setup parameters which are specific to a helicopter.");
+    return tr("Heli Setup is for Copter 4.0 or later.");
```

Comment:
The Heli page is not added unless the version is 4.0 or greater. So this isn't needed.

Reply:
@DonLakeFlyer I rebased on master, reverted that, squashed it and forced pushed it. Looked like a good idea at the time  :-)